### ISSUE TITLE
List unread messages

### ISSUE THREAD

**Issue by GoogleCodeExporter** [baluragala/gtalksms#53]
```
It'd be lovely to have a command that listed unread messages.


```

Original issue reported on code.google.com by `Eckankar` on 19 Jan 2011 at 2:54

**Comment by GoogleCodeExporter** [comment]
Original comment by `Florent....@gmail.com` on 19 Jan 2011 at 8:15
- Added labels: **Priority-Low**, **Type-Enhancement**
- Removed labels: **Priority-Medium**, **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
New command to read only unread messages : sms:unread
```

Original comment by `Florent....@gmail.com` on 22 Jan 2011 at 4:12
- Changed state: **WaitingForRelease**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 1.7.3
```

Original comment by `Florent....@gmail.com` on 22 Jan 2011 at 8:07
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 25 Aug 2011 at 7:51
- Changed state: **Deployed**
